.github/workflows/ci.yml

```diff
@@ -1,6 +1,6 @@
 name: Java CI
 
-on: [push, pull_request]
+on: [push]
```

Comment:
this configuration should be `on: [push, pull_request]`. Let's change it to this look